### PR TITLE
Session, Parser, CreateProject files and tests for Session and Parser ready for review  

### PR DESCRIPTION
**Pull request by ovinis**

### CODE REVIEW

File: tests/test_parser_delete.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+import argparse
+from pythontabcmd2.parsers.delete_parser import DeleteParser
+
+
+class DeleteGroupParserTestT(unittest.TestCase):
```
Comment (jacalata) [inline tests/test_parser_delete.py:12]: class name is wrong? file name should be more specific

File: pythontabcmd2/tabcmd2_controller.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import argparse
+import sys
+from .parsers.parent_parser import ParentParser
+from .logger_config import get_logger
+
+
+logger = get_logger('pythontabcmd2.tabcmd2_controller', 'info')
+
+
+class Tabcmd2Controller:
```
Comment (jacalata) [inline pythontabcmd2/tabcmd2_controller.py:10]: is there a specific reason this is in a separate class to main()? Seems like they could be combined for a tiny bit less complexity in reading the code.

File: pythontabcmd2/__init__.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+from .parsers.login_parser import LoginParser
```
[the remark this answers is not in the document]
Comment (jacalata) [reply pythontabcmd2/__init__.py:1]: Or perhaps we can get rid of the list altogether with a generic import from package https://docs.python.org/3/tutorial/modules.html#importing-from-a-package

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,20 @@
+*.pyc
+*.pkl
+*.log
+*.vscode
+*.csv
+*.pytest_cache
+
```
Comment (jacalata) [inline .gitignore:7]: add .DS_Store

File: pythontabcmd2/commands/project/publish_samples_command.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+            PublishSamplesParser.publish_samples_parser()
+        return cls(args, evaluated_project_path)
+
+    def run_command(self):
+        session = Session()
+        server_object = session.create_session(self.args)
+        self.publish_samples(server_object)
+
+    def publish_samples(self, server):
+        """Method to publish samples using tableauserverclient methods"""
```
Comment (jacalata) [inline pythontabcmd2/commands/project/publish_samples_command.py:32]: Does this one not actually do anything?

**Comment by jacalata** [comment]
@ovinis is this merge request still relevant or should I just close it?

**Comment by ovinis** [comment]
@jacalata We can close this and create a new PR for the release.

**Pull request closed without merging**